### PR TITLE
feat: Livestreamfails resolver

### PR DESCRIPTION
**Pull request by KararTY**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Added a Livestreamfails resolver for livestreamfails links:

https://livestreamfails.com/clip/109643 
image

NSFW marked clips will not show thumbnail:
https://livestreamfails.com/clip/109660
image

### CODE REVIEW

**Comment by KararTY** [comment]
On second note, I'm putting this pr into draft because lsf might change their CDN again:
image

File: internal/resolvers/livestreamfails/model.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		IsNSFW         bool       `json:"isNSFW"`
+		Label          string     `json:"label"`
+		SourceID       string     `json:"sourceId"`
+		SourceLink     string     `json:"sourceLink"`
+		SourcePlatform string     `json:"sourcePlatform"`
+	} `json:"streamer"`
+	StreamerID int       `json:"streamerId"`
+	UpdatedAt  time.Time `json:"updatedAt"`
+	VideoId    string    `json:"videoId"`
+}
```
Comment (zneix) [inline internal/resolvers/livestreamfails/model.go:70]: I believe that some fields of this struct may be unnecessary, you could try to remove some.

**Review by zneix — CHANGES_REQUESTED** [review]
Looks good to me

**Review by zneix — APPROVED** [review]

**Review by pajlada — CHANGES_REQUESTED** [review]

File: internal/resolvers/livestreamfails/load.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	}
+
+	// Build tooltip data from the API response
+	data := TooltipData{
+		NSFW:         clipData.IsNSFW,
+		Title:        clipData.Label,
+		Category:     clipData.Category.Label,
+		RedditScore:  clipData.RedditScore,
+		Platform:     strings.Title(strings.ToLower(clipData.SourcePlatform)),
+		StreamerName: clipData.Streamer.Label,
```
[the remark this answers is not in the document]
Comment (pajlada) [reply internal/resolvers/livestreamfails/load.go:61]: Nevermind, not necessary as you use the `html/template` package instead of `text/template`

**Review by pajlada — APPROVED** [review]

**Pull request merged**